packages/cli/core/README.md

```diff
@@ -1043,6 +1044,65 @@ EXAMPLES
 
 _See code: [src/commands/ui/create/vue.ts](https://github.com/coveo/cli/blob/@coveo/cli@2.2.3/packages/cli/core/src/commands/ui/create/vue.ts)_
 
+## `coveo ui:deploy`
+
+Deploy your search application to the Coveo infrastructure.
+
+```
+USAGE
+  $ coveo ui:deploy [-p <value>] [-c <value>] [-o <value>]
+
+FLAGS
+  -p, --pageId=7944ff4a-9943-4999-a3f6-3e81a7f6fb0a  The hosted page ID.
+  -c, --config=./configs/myconfig.json [default: coveo.deploy.json] The path to the configuration file.
+  -o, --organization=myOrgID  The identifier of the organization inside which to perform operations. See
+                              <https://docs.coveo.com/en/1562/#organization-id-and-other-information>.
```

Comment:
If we don't specify an organization, which one will be used?

Reply:
The one you're logged in w/ the cli